src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+  const setOfUniqueValues = new Set();
+  const arrayOfValues = Object.values(robot);
+
+  arrayOfValues.forEach((elem) => {
+    setOfUniqueValues.add(elem);
+  });
```

Comment:
set make them unique itself, no need to indicate it in its name

```suggestion
  const values = Object.values(robot);
  const uniqueValues = new Set(values);
```